chpl/chpl-service/src/test/java/gov/healthit/chpl/validation/listing/reviewer/FieldLengthReviewerTest.java

```diff
@@ -91,4 +94,26 @@ public void testBadDeveloperMaxLength_HasErrors_Pending() {
         assertTrue(listing.getErrorMessages().contains(BAD_LENGTH_DEVELOPER_PHONE));
         assertTrue(listing.getErrorMessages().contains(BAD_LENGTH_DEVELOPER_ZIP));
     }
+    
+    @Test
+    public void testBadTargetedUserMaxLength_HasErrors_Pending() {
+        PendingCertifiedProductDTO listing = mockUtil.createPending2014Listing();
+        PendingCertifiedProductTargetedUserDTO tu = new PendingCertifiedProductTargetedUserDTO();
+        tu.setName("This name is way too long.This name is way too long.This name is way too long.This name is way too long.This name is way too long.This name is way too long.This name is way too long.This name is way too long.This name is way too long.This name is way too long.This name is way too long.This name is way too long.This name is way too long.This name is way too long.This name is way too long.");
+        listing.getTargetedUsers().add(tu);
+        fieldLengthReivewerPending.review(listing);
+        assertTrue(listing.getWarningMessages().contains(BAD_LENGTH_TARGETED_USER));
+        assertTrue(listing.getTargetedUsers().isEmpty());
+    }
+    
+    @Test
+    public void testTargetedUserMaxLength_HasNoErrors_Pending() {
+        PendingCertifiedProductDTO listing = mockUtil.createPending2014Listing();
+        PendingCertifiedProductTargetedUserDTO tu = new PendingCertifiedProductTargetedUserDTO();
+        tu.setName("This name is not too long");
+        listing.getTargetedUsers().add(tu);
+        fieldLengthReivewerPending.review(listing);
+        assertFalse(listing.getWarningMessages().contains(BAD_LENGTH_TARGETED_USER));
```

Comment:
Thanks for adding tests! If there were to be a warning about this targeted user, it would not be the one being tested for.. it would be something like "You have exceeded the max length, 300 characters, for the Targeted User This name is not too long, which has been deleted". Can you change to test for the actual expected warning message?